pkgs/development/python-modules/argparse_addons/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "argparse_addons";
+  version = "0.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-uwiBB5RNM56NLnCnYwXd41FUTixb3rrxwttWrS5tzeg=";
+  };
+
+  pythonImportsCheck = [ "argparse_addons" ];
```

Comment:
Are we running the tests https://github.com/eerimoq/argparse_addons/tree/master/tests ?